Generate.py

```diff
@@ -133,7 +132,10 @@ def main(args=None) -> Tuple[argparse.Namespace, int]:
                     if yaml is None:
                         logging.warning(f"Ignoring empty yaml document #{doc_idx + 1} in {fname}")
                     else:
-                        weights_for_file.append(yaml)
+                        quantity = yaml.get("quantity", 1) if allow_quantity else 1
+
+                        for i in range(quantity):
```

Comment:
```suggestion
                        for _ in range(quantity):
```

potentially could also just use `weights_for_file += [yaml] * quantity`
tho I don't think that's necessarily better